README.md

```diff
@@ -1,42 +1,43 @@
-| Linkless Quest Support      | ✅                               | 🚧[Heavily delayed](https://discord.com/invite/cRdBUaUzcx)      | 🚧In development             |
-| Vr Multiplayer              | ⛔                               | ✅                             | 🚧In development             |
-| Spigot Plugin               | N/A                             | ✅                             | ⛔                            |
-| Patch Method                | N/A                             | Jar mod                       | Mixins                       |
-| Sponsored by Facebook       | Yes                             | No                            | No                           |
-| Initial Release Year        | 2016                            | 2013 / 2016                   | 2021                         |
-| Linux Support                 | ⛔                               | ✅                            | ✅                            |
-| Mac Support                 | ⛔                               | ✅?                            | ⛔                            |
+# Installation
+
+Installing the mod requires an understanding of programming in Java and managing natives. If you don't know what that means I recommend waiting until we start releasing alpha builds.<br>
+Once built, an OpenXR loader needs to be put in the `mods` folder, windows binaries of which can be retrieved from [here](https://nuget.org/packages/OpenXR.Loader)
+
+## Related Links
+
+Join the Discord server [here.](https://discord.gg/fyBye2ptkS)<br>
+Alternatively, join the Subreddit [here.](https://reddit.com/r/MinecraftXR)
+
+## Comparison Table
+
+|                              | [Official Minecraft VR](https://minecraft.net/en-us/vr) | [Vivecraft](http://vivecraft.org)                                       | [MCXR](https://github.com/Sorenon/MCXR)     |
+| ---------------------------- | ------------------------------------------------------- | ----------------------------------------------------------------------- | ------------------------------------------- |
+| State                        | Released                                                | Released                                                                | Pre-Alpha                                   |
+| Platform                     | Bedrock                                                 | Java                                                                    | Java                                        |
+| Mod Support                  | Bedrock Add-Ons                                         | Forge                                                                   | Fabric                                      |
+| Performance                  | :1st_place_medal:                                       | :2nd_place_medal:                                                       | :3rd_place_medal: (With Sodium)             |
```

Comment:
I think you misplaced medals for Vivecraft and MCXR

Reply:
Fixed!